charts/argo-templates/scripts/submit-file-ingest.py

```diff
@@ -1,25 +1,33 @@
+import os
+
 import google.auth
 from google.auth.transport.requests import AuthorizedSession
 from requests.exceptions import HTTPError
-import os
-
-credentials, project = google.auth.default(scopes=['openid', 'email', 'profile'])
-
-base_url = os.environ["API_URL"]
-dataset_id = os.environ["DATASET_ID"]
-profile_id = os.environ["PROFILE_ID"]
-source_path = os.environ["SOURCE_PATH"]
-target_path = os.environ["TARGET_PATH"]
 
-authed_session = AuthorizedSession(credentials)
 
-
-def ingest_file(dataset_id: str, **kwargs):
-    response = authed_session.post(f"{base_url}/api/repository/v1/datasets/{dataset_id}/files", json=kwargs)
+def ingest_file(session, base_url: str, dataset_id: str, profile_id: str, source_path: str, target_path: str):
+    response = session.post(f"{base_url}/api/repository/v1/datasets/{dataset_id}/files", json={
+        "profileId": profile_id,
+        "source_path": source_path,
+        "target_path": target_path
+    })
     if response.ok:
         return response.json()["id"]
     else:
         raise HTTPError(f"Bad response, got code of: {response.status_code}")
 
-# print the job id to std out
-print(ingest_file(dataset_id, profileId=profile_id, source_path=source_path, target_path=target_path))
+
+def authed_session():
+    credentials, project = google.auth.default(scopes=['openid', 'email', 'profile'])
+    return AuthorizedSession(credentials)
+
+
+if __name__ == '__main__':
```

Comment:
I'm interested to see if adding this will break how Argo tries to use this; I sure hope it doesn't but we'll see lol